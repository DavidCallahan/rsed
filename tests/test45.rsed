x = true
print $x
print not $x
y = false
print $y
print not $y
print $x < $y
print $x > $y
if $x == true then
   print "is true"
end
if (not $y) == true then
   print "is not true"
end
