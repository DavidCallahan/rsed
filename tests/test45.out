true
false
false
true
false
true
is true
is not true
